persistence/src/main/scala/com/lightbend/lagom/internal/persistence/PersistentEntityRegistryImpl.scala

```diff
@@ -7,6 +7,7 @@ import java.util.Optional
 import java.util.UUID
```

Comment:
change package to `com.lightbend.lagom.internal.persistence.cassandra`, where other Cassandra specific classes are located